lopezMidterm/src/App.jsx

```diff
@@ -1,17 +1,17 @@
 //import React from 'react';
-import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
-import Home from '/components/Home';
-import Products from '/components/Products';
-import ProductDetails from '/components/ProductDetails';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Home from '../components/Home.jsx';
+import Products from '../components/Products.jsx';
+import ProductDetails from '../components/ProductDetails.jsx';
 
 function App() {
   return (
     <Router>
-      <Switch>
-        <Route exact path='/Home' component={Home} />
-        <Route exact path="/Products" component={Products} />
-        <Route exact path="/Products/:id" component={ProductDetails} />
-      </Switch>
+      <Routes>
+        <Route exact path='/' element={Home} />
```

Comment:
One of the issues you are running into is caused here. Routes expect a component to be passed to them which means you need to include the braces too.

`<Route exact path='/' element={<Home />} />`